packages/cdk8s-plus-17/src/statefulset.ts

```diff
@@ -0,0 +1,189 @@
+import { ApiObject, ApiObjectMetadataDefinition, Lazy, Names } from 'cdk8s';
+import { Construct } from 'constructs';
+import { Resource, ResourceProps } from './base';
+import { Container, ContainerProps } from './container';
+import * as k8s from './imports/k8s';
+import { RestartPolicy, PodTemplate, IPodTemplate, PodTemplateProps } from './pod';
+import { Service } from './service';
+import { IServiceAccount } from './service-account';
+import { Volume } from './volume';
+
+
+export enum PodManagementPolicy {
```

Comment:
Add some doc strings. Should also be available in the k8s spec.

Reply:
Updated.